services/madoc-ts/src/frontend/site/pages/view-project.tsx

```diff
@@ -59,6 +60,14 @@ export const ViewProject: React.FC = () => {
 
       <SlotTabs initial={project?.isProjectMember ? 'project-my-work' : undefined}>
         <Slot name="project-navigation" label={t('Overview')} />
+          <MostRecentProjectUpdate />
+          <ListProjectUpdates />
```

Comment:
There is an "updates" slot/tab that we can put the full list in (and keep the "latest" on the overview)